packages/suite/src/components/suite/PrerequisitesGuide/PrerequisitesGuide.tsx

```diff
@@ -50,11 +52,11 @@ const BottomAnimatedContainer = styled(motion.div)`
     display: flex;
 `;
 
-const Bluetooth = () => (
+const Bluetooth = ({ children }: any) => (
```

Comment:
Rename & type pls.
(before it wasn't clear that it is gonna be a style wrapper, but now it is)
```suggestion
const BluetoothWrapper = ({ children }: PropsWithChildren) => (
```